LedRGB.cpp

```diff
@@ -36,8 +36,6 @@ void LedRGB::turnOff() {
     blueLed.turnOff();
 }
 
-void LedRGB::blink(unsigned n, unsigned delay1, unsigned delay2) {
-    redLed.blink(n, delay1, delay2);
-    greenLed.blink(n, delay1, delay2);
-    blueLed.blink(n, delay1, delay2);
+void LedRGB::rgbBlink(unsigned n, unsigned delay1, unsigned delay2) {
+    blink(n, delay1, delay2);
```

Comment:
This function call is much more complicated than it looks at the surface.  This is like something that would show up on an ecs40 exam and most people would get it wrong. I will do my best to explain why it does not work and what we can do to fix it at the meeting today.